tests/unit/configuration_subsystem/test_invalid_params.py

```diff
@@ -8,6 +8,7 @@
 
 from ansible_navigator.configuration_subsystem import NavigatorConfiguration
 from ansible_navigator.utils import ExitMessage
+
```

Comment:
I think we should use the option that drops the separator between the local and first-party imports.

Reply:
added: 
no_lines_before = "LOCALFOLDER"  # Keeps local imports bundled with first-party